### DISCUSSION TITLE
cron jobs work on windows but is this the right way?

### DISCUSSION THREAD

**Discussion by Mwendlin** [nilsteampassnet/TeamPass#4100] [Q&A]
Hello!
After much trial and error and with little knowledge, I have managed to get teampass running correctly on Windows and all tasks that require a cronjob seem to be executed correctly.

I have created a trigger in the task scheduler that is executed every 5 minutes.
This starts php.exe and as an argument I added every single file I found under teampass/scripts individually 

the whole thing works so far, but it is not quite clear to me whether each of these scripts is actually needed and whether it would have made more sense to create different triggers with different time intervals for each script.

maybe someone would like to advise me on this.
greetings Markus

**Comment by nelsondesfo** [comment]
I'm in the same situation. Some help?
